dbt_meshify/dbt_projects.py

```diff
@@ -328,11 +328,23 @@ def find_jinja_blocks(self) -> Dict[str, JinjaBlock]:
             )
 
         for unique_id, macro in self.manifest.macros.items():
+            block_type = "macro"
+            name = macro.name
+            top_level_folder = macro.path.split("/")[0]
+
             if macro.package_name != self.name:
                 continue
+            if (
+                top_level_folder in self.project.test_paths
+                if self.project.test_paths
+                else ["tests"]
+            ):
+                block_type = "test"
+                index = len(top_level_folder)
```

Comment:
How is `index` being used?

Reply:
ah wait that's wrong -- the `5` should still be hardcoded to remove `test_`from the name of the macro to properly search the jinja for `my_test` and not `test_my_test`

Reply:
Yes! Thanks makes sense 👍🏻